common/network/components/broadcaster.c

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2018 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include "broadcaster.h"
```

Comment:
Why didn't you include `common/network/components/broadcaster.h` for consistency?

Reply:
No particular reason, maybe same folder is smoother in case we change paths. But I agree for consistency.